src/main/java/net/pms/dlna/DLNAMediaInfo.java

```diff
@@ -1558,6 +1564,8 @@ public void postParse(int type, InputFile f) {
 				} else if (codecA.contains("pcm") || codecA.contains("wav") || codecA.contains("dts")) {
 					mimeType = HTTPResource.AUDIO_WAV_TYPEMIME;
 				} else if (codecA.contains("aac")) {
+					mimeType = HTTPResource.AUDIO_AAC_TYPEMIME;
```

Comment:
Lazzyness and the fact that it work as it, make stay away from touching it more and broken things.